src/pluginscript_callbacks.lua

```diff
@@ -61,7 +61,9 @@ local function print_coroutine_error(co, err, msg_prefix)
 		table_insert(msg_lines, table_concat(lps_callstack[i], ' '))
 	end
 	local msg = table_concat(msg_lines, '\n\tin ')
-	api.godot_print_error(msg, debug_getinfo(co, 0, 'n').name, filename, tonumber(line))
+	api.godot_print_error(
+		("%s:%d: %s"):format(filename, tonumber(line), msg:match("[%w%p%s]-%c"):gsub('%c', '', 1)), debug_getinfo(co, 0, 'n').name, filename, tonumber(line)
+	)
```

Comment:
Instead of matching the error string from `msg`, use `err_msg` directly instead, as it already contains the same error message. Don't forget to prepend the `msg_prefix`, something in the lines of `(msg_prefix or '') .. err_msg` should work.

Reply:
After that, since `msg` would not be used anymore, you should remove the lines that relate to the construction of `msg`. This includes the construction of `msg_lines` and the `for` loop that initializes the lines.

Reply:
Thank you, fixed it. Most of the time `msg_prefix` is nil, so I made it show the filename and line number in that case.